### DISCUSSION TITLE
Loading Spinner for lottie-react (Big JSON)

### DISCUSSION THREAD

**Discussion by ivanjeremic** [Gamote/lottie-react#23] [General]
I have a big Lottie file that lets my site load for a few seconds, I use Nextjs for this project and I would love to know if there is any way of displaying some JSX until Lottie has loaded the JSON?

I have tried to find an API in the Lottie-react docs but the docs are not very helpful and seem incomplete. Thanks

**Comment by Gamote** [comment]
Hi @ivanjeremic,

We currently don't have any functionality to handle the loading part. But this is a good suggestion. Currently I'm working on `v3` and I will definitely consider it.

**Reply by ivanjeremic** [reply]
Awesome can't wait for V3 :)

**Comment by RocketCodeMonkey** [comment]
Came here looking for this exact functionality, but realize that this comment is almost 3 years old and the package hasn't been updated in 7 months. Is this library still in current development, or has it fallen off the map?
